PsfLauncher/ErrorInformation.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+#include <string>
+#include <windows.h>
+#include <iostream>
+#include "utilities.h"
+
+class ErrorInformation
+{
+public:
+    ErrorInformation() : customMessage(L""), errorMessage(L""), isThereAnError(false)

```

Comment:
>customMessage(L""), errorMessage(L""), isThereAnError(false) [](start = 25, length = 60)

nit: use member initializers. E.g. `bool isThereAnError = false`. Also `std::basic_string` is initialized to an empty string by default; no need to explicitly do it